src/test/java/com/endava/service/IncomeExpenseServiceImplTest.java

```diff
@@ -293,13 +288,13 @@ private Wherefrom createWherefrom(String name) {
         return wherefrom;
     }
 
-    private Income createIncome(Double amount, Wherefrom wherefrom, Date date) {
-        Income income = new Income();
-        income.setAmount(amount);
-        income.setWherefrom(wherefrom);
-        income.setDate(date);
-        income.setUser(createUser());
+    private MoneyTransfer createMoneyTransfer(Double amount, Wherefrom wherefrom, Date date) {
+        MoneyTransfer moneyTransfer = new MoneyTransfer();
```

Comment:
Try to create inner static class Builder to avoid setters. Method chains are more readable (ex.):
return new MoneyTransfer.Builder()    // or add static import and : new Builder()
    .withAmount(amount)
    .withWherefrom(wherefrom)
    .withDate(date)
    .withUser(user)
    .withBlablabla(...)
    .build();